src/common/legend/style/legend.less

```diff
@@ -3,7 +3,7 @@
   right: 50px;
   left: auto;
   top: 0;
-  .z-index(@mapLevel);
```

Comment:
I'm not quite sure I understand what the previous syntax represented. I understand that the z-index controls the ordering, just not sure what @mapLevel does or why it was that previously.

Reply:
These are just manually set z-index levels behind the scenes.

```
/**
 * Z-index levels
 */
@mapLevel: 0.25;
@menuLevel: 0.5;
@dialogLevel: 0.75;
```

Reply:
All set in the same place so that they can pre-determine map, menu and dialog levels.

IMO: we should probably or create a new one for infobox at .4-ish and use menuLevel for the legend.

Reply:
@theduckylittle ^